CHANGELOG.md

```diff
@@ -88,6 +88,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2087](https://github.com/Pycord-Development/pycord/pull/2087))
 - Typehinted `command_prefix` and `help_command` arguments properly.
   ([#2099](https://github.com/Pycord-Development/pycord/pull/2099))
+- Support Annotated forms for typehinting slash command options
```

Comment:
This should be added
and under the added section

Reply:
Done.